packages/EasyErrorHandler/src/Bridge/Symfony/ErrorCodesProvider.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony;
```

Comment:
Please move to `EonX\EasyErrorHandler\Bridge\Symfony\Providers`

Reply:
Done.